Gemfile

```diff
@@ -40,7 +40,7 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "pry-byebug"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.5.0.beta4"
+  gem "rspec-rails", "~> 3.6.0.beta2"
```

Comment:
This solved the issue I ran into when trying to stub `policy`: https://github.com/rspec/rspec-mocks/pull/1104